src/index.ts

```diff
@@ -28,13 +28,20 @@ export class RequestError extends Error {
   }
 }
 
+/**
+ * Check if we're in a server-side rendering context
+ */
+const isSSR = () => typeof window === "undefined";
+
 /**
  * Client defines a client for interacting with the Sajari API.
  */
 export class Client {
   project: string;
   collection: string;
   endpoint: string;
+  key: string;
+  secret: string;
```

Comment:
I would just explictly mark these as optional, even though they have default values, just so that a user doesn't think they need to pass them through.
```suggestion
  key?: string;
  secret?: string;
```

Reply:
Good call. Done!